### PR TITLE
Add migration and modify init-codeamp-db.sh to include testdb

### PR DESCRIPTION
**Pull request by drshrey**

### CODE REVIEW

**Comment by drshrey** [comment]
addresses https://github.com/codeamp/circuit/issues/95

File: plugins/codeamp/migrations.go [hunk 1]
```diff
@@ -336,6 +336,24 @@ func (x *CodeAmp) Migrate() {
...
+				db.Find(&environments)
+				for _, env := range environments {
+					if env.Key == "" {
+						env.Key = env.Name
+					}
+					db.Save(&env)
+				}
+				return nil
+			},
+			Rollback: func(tx *gorm.DB) error {
```
Comment (sasso) [inline plugins/codeamp/migrations.go]: shouldn't this be key column?

Comment (sasso) [reply plugins/codeamp/migrations.go]: or empty because you didn't create anything here

Comment (drshrey) [reply plugins/codeamp/migrations.go]: yeah, it should be key. we need to drop the key column in rollback (since the migration is creating the key column)

**Pull request merged**